### PR TITLE
DOCSP-33784-capped-remediation

### PR DESCRIPTION
**Pull request by jason-price-mongodb**
Add small update for capped collections.

JIRA

https://jira.mongodb.org/browse/DOCSP-33784

STAGE

https://preview-mongodbjasonpricemongodb.gatsbyjs.io/cluster-sync/DOCSP-33784-capped-remediation/reference/disaster-recovery/#use-mongosync-for-disaster-recovery

BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=654589b2bd8a7a9579acc235

### CODE REVIEW

**Review by rohshar — CHANGES_REQUESTED** [review]
It seems that for every other change needed, it has its own section below the table as well. I would follow that same format.

File: source/reference/disaster-recovery.txt [hunk 1]
```diff
@@ -55,6 +55,10 @@ following changes:
        If synchronization was started with write blocking, disable the Write
        Block Mode.
 
+   * - :ref:`Capped Collections <manual-capped-collection>`
+     - During synchronization, capped collections are set to the maximum
+       allowable size. Use the ``collMod`` command to set your required
```
[the remark this answers is not in the document]
Comment (jason-price-mongodb) [reply source/reference/disaster-recovery.txt:60]: Will do. Was following the standard already set for all the other text in the section.

File: source/reference/disaster-recovery.txt [hunk 2]
```diff
@@ -55,6 +55,10 @@ following changes:
...
+       maximum size.
```
[the remark this answers is not in the document]
Comment (jason-price-mongodb) [reply source/reference/disaster-recovery.txt]: Done.

**Comment by jason-price-mongodb** [comment]
@rohshar please re-review. I've also added a new section per your request. Cheers.

File: source/reference/disaster-recovery.txt [hunk 3]
```diff
@@ -55,6 +55,8 @@ following changes:
        If synchronization was started with write blocking, disable the Write
        Block Mode.
 
+   * - :ref:`Capped Collections <c2c-dr-capped-collections>`
```
Comment (rohshar) [inline source/reference/disaster-recovery.txt:58]: I don't really want them to say the same thing. This should be a summary compared to below which has all of the information.

What about putting something like: "Synchronization sets capped collections to the maximum allowable size. Use the collMod command to resize capped collections"

**Review by rohshar — CHANGES_REQUESTED** [review]

**Comment by jason-price-mongodb** [comment]
@rohshar please re-review. Thanks.

File: source/reference/disaster-recovery.txt [hunk 4]
```diff
@@ -55,6 +55,10 @@ following changes:
...
+     - Synchronization sets capped collections to the maximum allowable
+       size. To set your required maximum size, use the ``collMod``
```
Comment (rohshar) [inline source/reference/disaster-recovery.txt:60]: I think this should be a link to collMod

**Review by rohshar — APPROVED** [review]
I'm LGTMing this, but please update the link before merging

**Pull request merged**